2 vs 3 and clap_derive/structopt

### Affected Version of clap

2.33.0 vs 3.0.0-alpha.1

### Expected Behavior Summary

`use clap::Clap` should work on latest version from `cargo`, b/c used in `README.md`.

### Reason for Behavior

`README.md` documents `3.0.0-alpha.1`, but `cargo` only has up through `2.33.0`.

### Proposed Fix

Add some sort of warning to the top of `README.md`, preferably with one or more of the following:

- a link to the `2.x` `README.md`
- a way to install `3.x` (a dependency with a git url?)

### Keywords (to make the issue easier to search for)

- ``error[E0432]: unresolved import `clap::Clap` ``
- ``no `Clap` in the root``
- ``error: cannot determine resolution for the derive macro `Clap` ``
- ``note: import resolution is stuck, try simplifying macro imports``
- ``error: cannot find attribute `clap` in this scope``
- ``error[E0599]: no function or associated item named `parse` found for struct `Opts` in the current scope``
- ``function or associated item `parse` not found for this``
- ``function or associated item not found in `Opts` ``

--- a/README.md
+++ b/README.md
@@ -17,7 +17,7 @@ It is a simple-to-use, efficient, and full-featured library for parsing command 
 > **IMPORTANT!**
 > This readme describes `clap v3.0.0` which is still under development and will be published soon-ish. 
 > It was not published on crates.io yet, and if you want to use this preview anyway, use 
-> ```toml
+> ```toml,no_sync
 > clap = { git = "https://github.com/clap-rs/clap/" }
 > ```
 > 

--- a/README.md
+++ b/README.md
@@ -14,6 +14,16 @@ It is a simple-to-use, efficient, and full-featured library for parsing command 
 * [website](https://clap.rs/)
 * [video tutorials](https://www.youtube.com/playlist?list=PLza5oFLQGTl2Z5T8g1pRkIynR3E0_pc7U)
 
+> **IMPORTANT!**
+> This readme describes `clap v3.0.0` which is still under development and will be published soon-ish. 
+> It was not published on crates.io yet, and if you want to use this preview anyway, use 
+> ```toml
+> clap = { git = "https://github.com/clap-rs/clap/" }
+> ```
+> 
+> If you're looking for the readme for `clap v2.33` - find it on [github](https://github.com/clap-rs/clap/tree/v2.33.0), [crates.io](https://crates.io/crates/clap/2.33.0), [docs.rs](https://docs.rs/clap/2.33.0/clap/).
+
+
 Table of Contents
 =================
 

